src/components/FlagSelect.tsx

```diff
@@ -1,3 +1,89 @@
-export function Welcome() {
-    return <h1>something</h1>;
-  }
\ No newline at end of file
+import React, { useEffect, useState } from "react";
+import { makeStyles } from "@material-ui/core/styles";
+import List from "@material-ui/core/List";
+import ListItem from "@material-ui/core/ListItem";
+import DialogTitle from "@material-ui/core/DialogTitle";
+import Dialog from "@material-ui/core/Dialog";
+import Australia from "/home/ishfaq/training-app/src/flags/Australia.png"
+import america from "/home/ishfaq/training-app/src/flags/america.png"
+import { Typography } from "@material-ui/core";
+
+const flags = {
+  imgs: [america, Australia],
+  name: ["+1-United States", "+61-Australia"]
+}
+const useStyles = makeStyles({
+  flags: {
+    paddingLeft: "12px"
+
+  }
+});
+
+export interface SimpleDialogProps {
+  open: boolean;
+  selectedIndex: number;
+  onClose: (selectedIndex: number) => void;
+}
+
+function SimpleDialog(props: SimpleDialogProps) {
+  const classes = useStyles();
+  const { onClose, selectedIndex, open } = props;
+
+  const handleClose = () => {
```

Comment:
you could use it like this
const handleClose = () => onClose(selectedIndex);
ES6 style

Reply:
it can be used if you have just one line of code for a method